apm-lambda-extension/extension/process_env.go

```diff
@@ -60,8 +61,14 @@ func getIntFromEnv(name string) (int, error) {
 func ProcessEnv() *extensionConfig {
 	dataReceiverTimeoutSeconds, err := getIntFromEnv("ELASTIC_APM_DATA_RECEIVER_TIMEOUT_SECONDS")
 	if err != nil {
-		log.Printf("Could not read ELASTIC_APM_DATA_RECEIVER_TIMEOUT_SECONDS, defaulting to 15: %v\n", err)
 		dataReceiverTimeoutSeconds = 15
+		log.Printf("Could not read ELASTIC_APM_DATA_RECEIVER_TIMEOUT_SECONDS, defaulting to %d: %v\n", dataReceiverTimeoutSeconds, err)
+	}
+
+	dataForwarderTimeoutSeconds, err := getIntFromEnv("ELASTIC_APM_DATA_FORWARDER_TIMEOUT_SECONDS")
+	if err != nil {
+		dataForwarderTimeoutSeconds = 3
```

Comment:
Maybe we should have these defaults as constants? Otherwise we'll have to search in the code for the place where we set the default.

Reply:
Solved in [d9a8edc](https://github.com/elastic/apm-aws-lambda/pull/136/commits/d9a8edc0ca29cd924a0217ed22f9715a5c270c90)